### PR TITLE
Show all asset from registry

### LINKED ISSUE #300: "Show all assets" button not working
Steps to reproduce:
Basilisk Rococo testnet -> Wallet page -> click on "Show all assets"

Expected:
All assets from the asset registry are shown.

Actual:
Only owned assets are shown in both active and inactive button state.

### PR DESCRIPTION
**Pull request by vkulinich-cl**
Fixes #300 
- Fix styling for the redeposited farms on mobile view
- Disable Transfer and Set fee asset payment buttons for the asset without balance

### CODE REVIEW

**Comment by green-jay** [comment]
button should be off by default (all assets not shown)

**Comment by jak-pan** [comment]
Is this g2g?

**Comment by vkulinich-cl** [comment]
> Is this g2g?

I'd say yes

**Review by jak-pan — APPROVED** [review]

**Comment by jak-pan** [comment]
Actually. The assets should have been shown because you can buy them from this screen.

**Comment by jak-pan** [comment]
Can u pls change it back and merge?

File: src/sections/wallet/assets/table/WalletAssetsTable.utils.tsx [hunk 1]
```diff
@@ -140,7 +140,11 @@ export const useAssetsTable = (
             actions.onAddLiquidity(row.original.id)
           }
           toggleExpanded={() => row.toggleExpanded()}
-          onTransferClick={() => actions.onTransfer(row.original.id)}
+          onTransferClick={
```
Comment (tomashapl1) [inline src/sections/wallet/assets/table/WalletAssetsTable.utils.tsx:143]:
```
() => {
    if(!row.original.total.isZero()) {
        actions.onTransfer(row.original.id)
    }
}
```

**Review by tomashapl1 — CHANGES_REQUESTED** [review]
Just small change, rest is OK 👏

**Comment by green-jay** [comment]
> Actually. The assets should have been shown because you can buy them from this screen.

based it on current prod hdx ui. should be aligned in either case imho

**Review by tomashapl1 — APPROVED** [review]

**Pull request merged**